solutions/ceoi-19-AmusementPark.mdx

```diff
@@ -0,0 +1,112 @@
+---
+id: ceoi-19-AmusementPark
+source: CEOI 2019
+title: 2019 - Amusement Park
+author: Shijie Ren
+---
+
+[CF Editorial](https://codeforces.com/blog/entry/68748)
+
+<Spoiler title="Hint">
+
+We can observe that for each legal proposal , reverse direction of every edges ,
+then we'll get another legal proposal.
+
+So the sum of costs will be $m\cdot\texttt{DAGCount}$.

```

Comment:
what does this have to do with reversing direction? Isn't it quite obvious that the sum of costs = number of configs * m?

Reply:
costs of two "opposite" valid proposals = m
so sum of costs = DAGcount / 2 * m
(Forgive my ignorance I don't think it's obvious)

Reply:
hmm.. maybe emphasize the two "opposite" valid proposals = m part in your solution

Reply:
Okay